tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -714,16 +714,18 @@ def __init__(self, nargs=0, **kw):
         super().__init__(nargs=nargs, **kw)
 
     def __call__(self, parser, namespace, values, option_string=None):
-        dep_msg = "Use of deprecated cli option {} detected. Option use in the following releases will be fatal. ".format(option_string)
+        dep_msg = "The {} option is deprecated and will be removed in the next release. ".format(option_string)
         if 'fusing' in option_string:
-            dep_msg += 'Please use --finegrain_fusing cli option instead'
+            dep_msg += 'Please use --finegrain_fusing instead.'
```

Comment:
`--finegrain_fusing` is also deprecated. 

```
    common_group.add_argument('--finegrain_fusing',
                              help='[DEPRECATED] Regex for layers/operations that won\'t be fused. ' +
                                   'Example: --finegrain_fusing Convolution1,.*Scale.*',
                              action=DeprecatedOptionCommon)
```

Please do not recommend this option. Just remove this `if`.

Reply:
Helena, I understand. It will be great to clean-up this as well:)